lib/facebook/messenger/server.rb

```diff
@@ -14,9 +14,8 @@ def call(env)
         @request = Rack::Request.new(env)
         @response = Rack::Response.new
 
-        case
-        when @request.get? then verify
-        when @request.post? then receive
+        if @request.get? then verify
+        elsif @request.post? then receive
```

Comment:
@jgorset Code wise, they both look the same except the extra case statement, same for readability. `Rubocop` now, doesn't like case.